### DISCUSSION TITLE
[Solved] Photos missing at Search compared to Calendar

### DISCUSSION THREAD

**Discussion by ant0nwax** [photoprism/photoprism#2304] [Q&A] [answered]
Hi,

Thanks first of all for this awesome piece of software, even if it nees a lot of ressources, i am still astonished what is possible on the open source market... I already considered donating, too.
I am quite new to PhotoPrism (I am an ex Google Photos User) and my background is I am System Engineer
I have a question according the Search (basic Home Page) of PhotoPrism

As you see on the two screenshots, in Search for May 2022 there are less photos than for Calnedar May 2022 
mostly important documents that I just uploaded are not on the search page, which would speed up my workflow if they all exist on the homepage.

this issue is maybe not always with text documents that are photographed as jpg, but also sometimes normal photos are missing at search.

as a workaround i could go to search and find the approx. time of similar photos of a day that i remember (often they are missing ALL photos of that day at search) and then i go to calendar and browse the whole month... quite slow process/workaround for me.

Is there a better solution? Maybe to see ALL photos at Search?

SEARCH:
![SearchCapture](https://user-images.githubusercontent.com/33480652/167075638-270a69c1-0b2e-46df-9de6-9098b1b2e196.PNG)

CALENDAR:
![CalendarCapture](https://user-images.githubusercontent.com/33480652/167075635-c70e9a7f-e5af-405c-bfc6-745a13e2bd18.PNG)

**Comment by graciousgrey** [comment, marked as answer]
I could reproduce this. I assume that the files that are shown in calendar and not in search are in review (https://docs.photoprism.app/user-guide/organize/review/). Once you approve them, they should show up in search as well.

**Comment by ant0nwax** [comment]
Thank you, I will learn about review approval process

the "scans" are photos with my mobile, some show up, some not

is there a way to batch approve everything?

**Comment by ant0nwax** [comment]
Hi @graciousgrey I deactivated Quality in Settings, thanks for your hint, it solved this issue, i will mark it solved

**Comment by jrstrunk** [comment]
Imo the Quality filter is a great feature for some users, but leaves a lot of other users confused. I think it should be off by default but able to be turned on by those who want it.
